cdap-api/src/main/java/co/cask/cdap/api/app/AbstractApplication.java

```diff
@@ -225,24 +227,25 @@ protected void addService(String name, HttpServiceHandler handler, HttpServiceHa
    * @param schedule the schedule to be added for the Workflow
    * @param workflowName the name of the Workflow
    */
+  @Deprecated
```

Comment:
Yes it works on ``TimeSchedule``. They're not that common. It's just that they are both schedules and the same operations are performed on them. Not too sure what you're getting at?